config/globals.js

```diff
@@ -2,4 +2,6 @@ module.exports = {
 	SC_THEME: process.env.SC_THEME || 'default',
 	DOCUMENT_BASE_DIR: process.env.DOCUMENT_BASE_DIR || 'https://s3.hidrive.strato.com/schul-cloud-hpi/',
 	DATABASE_AUDIT: process.env.DATABASE_AUDIT || 'false',
+	BODYPARSER_JSON_LIMIT: process.env.BODYPARSER_JSON_LIMIT || '20mb',
+	MAXIMUM_ALLOWABLE_TOTAL_ATTACHMENTS_SIZE: (5 * 1024 * 1024), // 5MB
```

Comment:
for values that are numbers, add the type always
```suggestion
	MAXIMUM_ALLOWABLE_TOTAL_ATTACHMENTS_SIZE_MB: (5 * 1024 * 1024), // 5MB
```